rerun_cpp/src/rerun/blueprint/components/active_tab.hpp

```diff
@@ -50,26 +44,23 @@ namespace rerun::blueprint::components {
 } // namespace rerun::blueprint::components
 
 namespace rerun {
-    template <typename T>
-    struct Loggable;
+    static_assert(sizeof(rerun::datatypes::EntityPath) == sizeof(blueprint::components::ActiveTab));
 
     /// \private
     template <>
     struct Loggable<blueprint::components::ActiveTab> {
         static constexpr const char Name[] = "rerun.blueprint.components.ActiveTab";
 
         /// Returns the arrow data type this type corresponds to.
-        static const std::shared_ptr<arrow::DataType>& arrow_datatype();
-
-        /// Fills an arrow array builder with an array of this type.
-        static rerun::Error fill_arrow_array_builder(
-            arrow::StringBuilder* builder, const blueprint::components::ActiveTab* elements,
-            size_t num_elements
-        );
+        static const std::shared_ptr<arrow::DataType>& arrow_datatype() {
```

Comment:
The PR removes `namespace arrow { class DataType; }` but still refers to `arrow::DataType`

Reply:
That was intentional: when we're forwarding we know that we include the target datatype which has to forward declare this. It's a bit philosophical though and I figure you'll easy find that we're not entirely consistent in assuming that another header drags something in or not. 🤔